NeoML/include/NeoML/Dnn/Layers/LstmLayer.h

```diff
@@ -39,13 +39,13 @@ class NEOML_API CLstmLayer : public CRecurrentLayer {
 	void SetHiddenSize(int size);
 
 	// The input hidden layers weights (the blob size is (4*HiddenSize)x1x1xInputSize)
-	CPtr<CDnnBlob> GetInputWeigthsData() const { return inputHiddenLayer->GetWeightsData(); }
+	CPtr<CDnnBlob> GetInputWeightsData() const { return inputHiddenLayer->GetWeightsData(); }
```

Comment:
This method is invoked by the python wrapper.

Reply:
Fixed Python wrapper too